src/test/java/racingcar/domain/CarsTest.java

```diff
@@ -0,0 +1,46 @@
+package racingcar.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+public class CarsTest {
+
+    @ParameterizedTest
+    @CsvSource(value = {"true:1", "false:0"}, delimiter = ':')
+    void 모든차앞으로전진(boolean moveNum, int expectedPosition) {
+        // given
+        Cars cars = Cars.of(Arrays.asList("qwe", "asdf", "zx"));
+
+        // when
+        cars.moveAll(()->moveNum);
```

Comment:
IDE의 자동 정렬 기능을 통해서 코드 정렬을 습관적으로 진행해주세요! 🙂
맥의 경우 `⌥ + ⌘ + L`의 단축키를 통해서 코드 정렬을 할 수 있어요! 🤗
```suggestion
        cars.moveAll(() -> moveNum);
```